Functions/CatalogItems/Write-RsCatalogItem.ps1

```diff
@@ -123,7 +124,7 @@ function Write-RsCatalogItem
                 $enabled = $content.DataSourceDefinition.Enabled
                 $credentialRetrieval = 'None'
 
-                $newDataSourceCmd = "New-RsDataSource -Destination $Destination -Name $itemName -Extension $extension -CredentialRetrieval $credentialRetrieval"
+                $newDataSourceCmd = "New-RsDataSource -RsFolder $RsFolder -Name $itemName -Extension $extension -CredentialRetrieval $credentialRetrieval"
```

Comment:
(Optional) Could you also update New-RsFolder to accept a RsFolder instead of Path as a parameter to the function?

---
In reply to: [101355543](https://github.com/Microsoft/ReportingServicesTools/pull/23#discussion_r101355543) [](ancestors = 101355543)